api.go

```diff
@@ -983,6 +984,60 @@ func (api *API) Info() serverInfo {
 	}
 }
 
+// EnsureOperatingNode determine if the current node is the destination for execution if not forward on to correct node
+func (api *API) EnsureOperatingNode(indexName string, w http.ResponseWriter, r *http.Request) bool {
```

Comment:
the comment on this method needs to explain what the return values mean. "true" meaning that the query did not belong to this node was unintuitive for me. I don't care which value means what though, I just want it documented.